### PR TITLE
Add k8s dashbard v1.6.2

### PR DESCRIPTION
**Pull request by Globegitter**

### CODE REVIEW

File: docs/addons.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The [dashboard project](https://github.com/kubernetes/dashboard) provides a nice
 
 Install using:
 ```
-kubectl create -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/kubernetes-dashboard/v1.6.1.yaml
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/kubernetes-dashboard/v1.6..yaml
```
Comment (justinsb) [inline docs/addons.md]: Accidentally removed the 2 :-)

**Comment by justinsb** [comment]
Thanks - lgtm except for the minor removal of the 2 in the docs/addons.md

**Comment by Globegitter** [comment]
@justinsb Just saw they already updated to 1.6.3, I might as well update straight to that version.

**Comment by justinsb** [comment]
/ok-to-test

/lgtm

Thanks @Globegitter !

**Pull request merged**